ports/cvtt/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "cvtt",
+  "version-date": "2022-06-08",
+  "description": "Fast, high-quality texture compression library for many formats ",
```

```suggestion
  "name": "convectionkernels",
  "version-date": "2022-06-08",
  "description": "Fast, high-quality texture compression library for many formats, also known as cvtt",
```
We strongly prefer names longer than 5 characters (unless there's significant prior art). I also get much better SEO in search engines for "convectionkernels" (top hit is this library) versus (cvtt).

Adding it to the description makes sure that `vcpkg search cvtt` will still find it.